github/with_appengine.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2017 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// This file provides glue for making github work on App Engine.
+
+// +build appengine
+
+package github
+
+import (
+	"context"
```

Comment:
That's an excellent point, and I believe this was an oversight on my part.
Internally, we have our own `context` so I didn't remember this was an issue.
I'll change it.